### ISSUE TITLE
utils.modules.Conv2d seems wrong

### ISSUE THREAD

**Issue by CollieX** [yjh0410/yolov2-yolov3_PyTorch#35]
As far as I know, a convolutional layer should not have bias when a Batch Normalization applied after it. I think it is a serious issue according to the theory of Batch Normalization. I would appreciate if you learn more about Batch Normalization and fix the issue.
Thus you can add a `bias=False` as a kwarg of `torch.nn.Conv2d` in `utils.modules.Conv2d` module, then retrain the models.

**Comment by yjh0410** [comment]
Even if I use bias in Conv2d, the bias will be eliminated by bn which means it is equal to 0 bias. But it doesn't mean we have to cancel bias in conv. I still can randomly set bias. So I won't retrain it. It is unnecessary.

---Original---

**Comment by CollieX** [comment]
Fine.But don't you think it is unnecessary?And add some redundant computation costs……

**Comment by yjh0410** [comment]
Yeah, I agree with you that useless bias will bring some redundant computation cost.
But, I have only one GPU... If I retrain yolov2 on coco(voc is too simple.), I have to wait for a half of a month....
So sorry~I have no choice~
All in all, thanks for your advice !(๑&gt;؂<๑）



---Original---

**Comment by CollieX** [comment]
Hello, I think you can name your lightweight model based on YOLO v2 as **Slim YOLO v2**.
By the way, I am going to study in the **Tsinghua Shenzhen International Graduate School (Tsinghua SIGS)** since September, major in Big Data.

**Comment by yjh0410** [comment]
Nice to see you again！
Slim YOLOV2？Sounds wonderful！
Recently, I'm trying to reproduce TinyYOLOv3.
And，congratulation！！
I am glad to hear the good news！&nbsp;
I sincerely wish you have a good time in Tsinghua SIGS.&nbsp;
By the way, I'm studying for a PhD in HIT.&nbsp;

---Original---
